pkg/netpol/connlist/conns_formatter.go

```diff
@@ -13,6 +15,22 @@ func getNewLineChar() string {
 	return fmt.Sprintln("")
 }
 
+// gets the conns array and returns a sorted array of singleConnFields struct. helps with forming the json and csv outputs
+func writeConnsIntoSortedSingleConnFieldsArray(conns []Peer2PeerConnection) []singleConnFields {
```

Comment:
The `write` verb does not reflect the actual operation done by the function. I think `sortConnections` may be more accurate